backend/api/server.js

```diff
@@ -104,18 +99,51 @@ server.post("/api/signup/", async (req, res, next) => {
 			},
 		})
 
-		const token = await registerSessionToken(data)
-		const clientResponseInformation = {
-			id: data.id,
-			accountType: data.accountType,
-			token: token,
+		const jwtToken = await registerSessionToken(createdAccount)
+		const clientAccountInformation = formatClientAccountInformation(
+			createdAccount,
+			jwtToken
+		)
+
+		return res.status(200).json(clientAccountInformation)
+	} catch (error) {
+		next(error)
+	}
+})
+
+server.post("/api/signup/team", async (req, res, next) => {
```

Comment:
You should authenticate each of these requests otherwise it's a security vulnerability. https://www.digitalocean.com/community/tutorials/nodejs-jwt-expressjs